Scripts/assignment/emme_bindings/mock_project.py

```diff
@@ -309,7 +309,7 @@ def congested_assignment(self, transit_assignment_spec, class_names,
             "TRANSIT_SEGMENT", "@base_timtr", "", 1.0,
             overwrite=True, scenario=scenario)
         report = {
-            "stopping_criterion": "MAX_ITERATIONS",
+            "stopping_criteria": "MAX_ITERATIONS",
```

Comment:
"criterion" is singular and "criterion" is plural, so this change should not be needed? I did a quick Find All for `olusanya` and both are used...

Reply:
Yes, but the most important task of `MockProject` is to produce the exact same result as EMME. 😄

Reply:
Ah okay, if it is in Emme then nothing! 😄